docs/api/graph/heatmap.md

```diff
@@ -0,0 +1,90 @@
+Atlas primarily supports visualizing data in line charts. As of 1.8, Atlas can also visualize
+via heatmaps using the [`:heatmap`](../../api/graph/line-styles.md#heat-map) line style.
+The graph area is broken up into a series of cells and a count for each cell is incremented when 
+a measurement falls within the cells boundaries. Colors or shades of colors then fill in cells 
+based on the final count.
+
+## Percentiles
+Heat maps are particularly useful on top of
+[percentile](../../spectator/patterns/percentile-timer.md) metrics to analyze the entire
+measurement range. 
+
+**Note** the query must be setup properly for percentile metrics. Instead of using the 
+`:percentile` operator, the query must group on the `percentile` tag via `(,percentile,),:by`.
+Using the [log linear](axis-scale.md#log-linear) scale will help to highlight clustered
+regions of measurements via `&scale=log-linear`.
+The example also uses data not available in the demo Atlas instance.
+
+@@@ atlas-uri { hilite=:heatmap }
+/api/v1/graph?e=2012-01-01T00:00&no_legend=1&q=name,requestLatency,:eq,(,percentile,),:by,:heatmap&scale=log-linear
```

Comment:
Users should just use `:percentiles-heatmap`:

```
name,requestLatency,:eq,:percentiles-heatmap
```

They don't need to worry about what it does.

Reply:
ah good, that's still in there. Will update.